README.md

```diff
@@ -109,7 +121,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+filename.jpg
```

Comment:
`filename.jpg`? srsly?

This is strange: your diagram has the correct topology, but it swaps state names for labels.

In a state machine you have states (the circles) and directed transitions between the circles. The transitions are each labelled with an event. When you are in a particular state and you receive an event, you look for the transition with that even as a label and follow it to the next state.

So there should be a transition from S0 to S1 labelled "The". And so on.

Here's my version:

![image](https://user-images.githubusercontent.com/10648/78728517-2f122a00-78fd-11ea-9379-a45fe6d1a3ee.png)

# -3